Source/ASRunLoopQueue.mm

```diff
@@ -238,14 +238,12 @@ - (void)releaseObjectInBackground:(id  _Nullable __strong *)objectPtr
 
 - (void)drain
 {
-  @autoreleasepool {
-    _lock.lock();
-    auto q = std::move(_queue);
-    _lock.unlock();
-    for (auto ref : q) {
-      // NOTE: Could check that retain count is 1 and retry later if not.
-      CFRelease(ref);
-    }
+  _lock.lock();
+  auto q = std::move(_queue);
+  _lock.unlock();
+  for (auto ref : q) {
+    // NOTE: Could check that retain count is 1 and retry later if not.
+    CFRelease(ref);
```

Comment:
I was silly for putting this in in the first place. Creating autorelease pools takes a lot of time and there's absolutely no chance of anything getting added to it in here 🤦‍♂️